README.Rmd

```diff
@@ -13,1044 +12,68 @@ knitr::opts_chunk$set(
 )
 ```
 
-# r2dii.climate.stress.test
+# r2dii.climate.stress.test <a href='https://github.com/2DegreesInvesting/r2dii.climate.stress.test'><img src='https://imgur.com/A5ASZPE.png' align='right' height='43' /></a>
 
 <!-- badges: start -->
+[![lifecycle](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://www.tidyverse.org/lifecycle/#experimental)
 [![R-CMD-check](https://github.com/2DegreesInvesting/r2dii.climate.stress.test/workflows/R-CMD-check/badge.svg)](https://github.com/2DegreesInvesting/r2dii.climate.stress.test/actions)
 <!-- badges: end -->
 
-Beta version. More soon…
-
-The goal of r2dii.climate.stress.test is to provide a tool that can be used to conduct what-if climate stress test analyses for financial institutions, supervisors, regulators and other stakeholders. The tool aims at highlighting potential financial risk in especially climate relevant sectors, split by production technology where required. The sectors covered by the 2Dii climate stress test and therefore by this package, follow mostly the logic of the Paris Agreement Capital Transition Assessment (PACTA) tool, but can in principle be adapted to other settings.
-W.I.P.
+The goal of r2dii.climate.stress.test is to provide a tool that can be used to conduct what-if climate stress test analyses for financial institutions, supervisors, regulators and other stakeholders. The tool aims at highlighting potential financial risk in especially climate relevant sectors, split by production technology where required. The sectors covered by the 2Dii climate stress test and therefore by this package, follow mostly the logic of the Paris Agreement Capital Transition Assessment (PACTA) tool, but can in principle be adapted to other settings. Application of the code requires availability of custom data. In order to participate in a climate stress testing project with 2 degrees investing initiative please contact jacob@2degrees-investing.org.
```

Comment:
I believe we normally use 2DII, not 2Dii.

Reply:
and I would add: for more information about the methodology and inquiries on running a pilot of the stress test in cooperation with 2DII, please contact ....

Reply:
DONE